### DISCUSSION TITLE
LEDVANCE SMART+ WiFi Classic 100 14W/2700K E27

### DISCUSSION THREAD

**Discussion by davidebeatrici** [arendst/Tasmota#15927] [Q&A] [answered]
Powered by a [WB8P](https://developer.tuya.com/en/docs/iot/wb8p-module-datasheet?id=K9fwx4f89tvzd), does a pin-compatible replacement exist?

<details>
<summary>Photos</summary>

Comparison against a "normal" bulb (right):

![](https://user-images.githubusercontent.com/5897523/177240195-0b628393-eb95-4222-bc47-a6cf66bef16c.png)

![](https://user-images.githubusercontent.com/5897523/177240948-b992554c-8190-42da-ae32-494b011c5e31.png)
![](https://user-images.githubusercontent.com/5897523/177240938-4ef461be-6224-494c-8d27-3688753c8cc1.png)
![](https://user-images.githubusercontent.com/5897523/177240942-e8d7dd6d-052b-4738-b439-b2a873e122f9.png)
![](https://user-images.githubusercontent.com/5897523/177240945-9ee2b282-9ea4-4886-8605-23605c0b3376.png)
</details>

---

[Amazon.it](https://www.amazon.it/gp/product/B08KYB6HD9)

[Official website](https://www.ledvance.com/consumer/products/smart-home/smart-lamps/smart-wifi/smart-classic-heatsink-lamps-with-wifi-technology/classic-bulb-shape-with-wifi-technology-c6445)

Datasheets:

[4058075485471_SMART__WiFi_Classic_100__14_W_2700_K_E27__en.pdf](https://github.com/arendst/Tasmota/files/9042046/4058075485471_SMART__WiFi_Classic_100__14_W_2700_K_E27__en.pdf)
[4058075485839_SMART__WiFi_Classic_100__14_W_2700_K_E27__en.pdf](https://github.com/arendst/Tasmota/files/9042047/4058075485839_SMART__WiFi_Classic_100__14_W_2700_K_E27__en.pdf)

The only difference between the two appears to be the packaging type.

**Comment by barbudor** [comment, marked as answer]
I don't recall having seen such form factor before

Most modules are listed here: https://tasmota.github.io/docs/Pinouts/

Transplant will require some imagination

**Reply by sfromis** [reply]
Here's a post about WB8P transplant:
https://www.digiblur.com/2021/02/treatlife-sl10-rgb-ccw-esp-transplant.html
